packages/cactus-plugin-something-somehow/package.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I misunderstood your proposal on this pull-request. It seems that our proposal is to move socket.io typed validators codes of the following codes to some subdirectory

examples/cartrade, electricity-trade, run-transaction, test-run-transaction
packages/business-logic-plugin, config, ledger-plugin, routing-interface, copyStaticAssets.ts, jest.config.js, package.json, tsconfig.json, tslint.json, script-build-packages.sh
but I cannot agree with your proposal because this proposal cannot take advantages of both types of openAPI-typed Validators and socket.io-typed validators.
I think your proposal means that duplicated package.json are inconvenient for CI checking system. But I'd like to keep the above codes on this repository in as easy-to-use form as the other codes. Would you mind if you give me some advice on this issues?

Reply:
@takeutak Sure thing, happy to provide advice! => Please break it[1] down to a list of objective points/criteria that can be addressed and/or measured in a non-subjective way. For example, the expression "easy-to-use" could be broken down to a list of problems that would make it not easy to use with the proposed directory package structure in this PR.

[1]: 
> But I'd like to keep the above codes on this repository in as easy-to-use form as the other codes.

Reply:
@petermetz Sorry to reply late. In order to decide them in subjective way, I'm sure that it is needed to merge the codes. As I already contacted, I'm planning to merge the codes with adapting Verifier to OpenAPI-typed Validator. Would you wait my answer until the development of adapting Verifier?

Reply:
> Would you wait my answer until the development of adapting Verifier like the above?

@takeutak I'd rather move this along as fast as possible to be honest. Could you please let me know:
What is the nature of the dependency between the two that makes the wait necessary? 
What is the reason for that PR having to jump the review queue (potentially putting the burden of merge on the other PRs like this one)?